### ISSUE TITLE
Handlebars errors on newsletter generation

### ISSUE THREAD

**Issue by slifty** [TechAndCheck/tech-and-check-alerts#412]
I [merged in](https://github.com/TechAndCheck/tech-and-check-alerts/pull/399) a handlebars version upgrade without fully testing; shame on me.

It's generating errors now:

```
Handlebars: Access has been denied to resolve the property "speakerNormalizedName" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```

The error [links to](https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access) a page talking a bit more about how to get around the issue; let's either fix the access to be safe OR disable the warning.
